### DISCUSSION TITLE
RainbowMiner does not detect my AMD RX5700XT graphics card

### DISCUSSION THREAD

**Discussion by thiagoreissantos** [RainbowMiner/RainbowMiner#2530] [Q&A]
I'm running Ubuntu 22.04 LTS. I have two Nvidia RTX 3070 Ti cards that are mining normally. After a while, I added an AMD RX5700XT card to my system.

I followed the tutorial to install the AMD drivers. I was able to install normally. I ran the rocm-smi command and the system displayed the AMD card information normally.

However, RainbowMiner was unable to detect the AMD card, so I am not able to mine with the new RX5700XT card.

I will be grateful for any help.

**Comment by RainbowMiner** [comment]
Please upload a Debug file (click on "Debug file" on http://localhost:4000).

**Reply by thiagoreissantos** [reply]
Thanks for the quick response. Attached are the log files.

[Logs.zip](https://github.com/RainbowMiner/RainbowMiner/files/13369638/Logs.zip)
